components/prompt_based_laion_retrieval/src/main.py

```diff
@@ -29,7 +29,10 @@ def query_clip_client(text: str, client: ClipClient) -> List[str]:
         urls: a list of strings, each representing a URL of an image related to the query
     """
     results = client.query(text=text)
-    urls = [i["url"] for i in results]
+    try:
+        urls = [i["url"] for i in results]
+    except:
+        urls = ["" for _ in range(len(results))]
```

Comment:
Why is this try except needed? Would prefer to at least not have a bare `except` (and I'm surprised ruff doesn't complain about this)